graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -542,15 +548,22 @@ def _rename_or_suppress_or_ignore_name_and_add_to_record(
         if (
             new_name_string in self.reverse_name_map
             and self.reverse_name_map[new_name_string] != name_string
```

Comment:
Technically in the scope of the PR, since you're modifying this conditional — this can be more succinctly and more clearly stated as `self.reverse_name_map.get(new_name_string, None) != name_string`. Since an easy alternative like this is available, not taking it tricks the reader into thinking something subtle and unusual is going on — I had to double-check, for example, that the same `self.reverse_name_map` dict was used in both clauses.

I'd much prefer the `.get()` approach here as a result.

Reply:
This is actually a bit trickier than it looks on the surface, since we're checking if there already is a mapping for `new_name_string` that doesn't correspond to our current `name_string`, i.e. there's a conflict. The code that makes this work is `self.reverse_name_map.get(new_name_string, name_string) != name_string`, i.e. the default is `name_string` instead of `None`. I don't have a strong preference since I've seen this code a whole bunch of times, but would you say one seems more understandable than the other?

Reply:
I'd definitely prefer the code one-liner + some explanation in English, over the current two-clauses-on-two-lines approach. I think it'd be much more readable.

Reply:
Especially because it's trickier than it looks on the surface :D